### PR TITLE
global: upgrade invenio-app-ils

### PR DESCRIPTION
**Pull request by kpsherva**
* search: remove es6 compatibility
* closes https://github.com/inveniosoftware/invenio-app-ils/issues/1188

### CODE REVIEW

File: requirements.pinned.txt [hunk 1]
```diff
@@ -1,177 +1,185 @@
...
-future==0.18.2
+Flask-WTF==1.2.1
+fs==2.4.16
+ftfy==6.1.3
+future==0.18.3
 fuzzywuzzy==0.18.0
-idna==3.4
-importlib-metadata==4.8.3
-importlib-resources==5.4.0
+github3.py==4.0.1
```
Comment (ntarocco) [inline requirements.pinned.txt:60]: We should not need `github`?

File: requirements.pinned.txt [hunk 2]
```diff
@@ -1,177 +1,185 @@
...
+greenlet==3.0.3
```
Comment (ntarocco) [inline requirements.pinned.txt:61]: are we sure about `greenlet`?

**Review by ntarocco — APPROVED** [review]

File: ui/package.json [hunk 3]
```diff
@@ -6,7 +6,9 @@
     "@inveniosoftware/react-invenio-app-ils": "^1.0.0-alpha.77",
     "@rjsf/core": "^2.5.0",
     "@rjsf/semantic-ui": "^2.5.0",
-    "axios": "^0.21.0",
+    "ajv": "^8.0.0",
+    "ajv-keywords": "^5.0.0",
+    "axios": "^1.6.0",
```
[the remark this answers is not in the document]
Comment (kpsherva) [reply ui/package.json:11]: it is working fine, I tested GET and POST

File: requirements.pinned.txt (hunk 1 above)
Comment (kpsherva) [reply requirements.pinned.txt:60]: invenio-oauthclient requires it :)

File: requirements.pinned.txt (hunk 2 above)
Comment (kpsherva) [reply requirements.pinned.txt:61]: thats a good question, it gets installed with some other dependency, but pip does not want to disclose which dependency installs it:
Screenshot 2024-01-19 at 17 21 41

**Pull request merged**